src/_includes/macros/activities-block.html

```diff
@@ -0,0 +1,32 @@
+
+{# ==========================================================================
+
+   activities_block.render()
+
+   ==========================================================================
+
+   Description:
+
+   Render activities block when given:
+
+   tags: List of activities tags.
+
+   ========================================================================== #}
+
+{% macro render(tags) %}
+{% from 'macros/util/text.html' import string_length as string_length %}
+<section class="block
+                block__bg
+                block__flush-sides
+                block__flush-top
+                block__flush-bottom
+                qa-tags">
+    {% import 'macros/activity-snippets.html' as activity_snippets with context %}
+    {% set activities_feed = activity_snippets.render(tags, include_date_flag=true, number_columns=2) %}
+    {% include 'templates/activities-feed.html' %}
+    <a class="jump-link jump-link__underline"
+       href="/activity-log/{{ ('?filter_tags=' ~ tags | join('&') ) if tags }}">
+        View all of our activities
```

Comment:
Seems like it would make sense to add that at the point we need to override the text. Right now I think offices and subpages share the same text.